doc/tor.md

```diff
@@ -146,6 +145,53 @@ password` (refer to the [Tor Dev
 Manual](https://2019.www.torproject.org/docs/tor-manual.html.en) for more
 details).
 
+## Example: Bitcoin Core Onion Service on Ubuntu 20.04.2 LTS
```

Comment:
Changed header. Maybe some content can be changed or rephrased but one example is required. I was considering adding Windows and Android with screenshots as well but then dropped the idea because others may not agree to it.